core/src/main/java/io/grpc/CallCredentials.java

```diff
@@ -65,6 +72,7 @@
    * @param applier The outlet of the produced headers. It can be called either before or after this
    *        method returns.
    */
+  @ExperimentalApi("https//github.com/grpc/grpc-java/issues/1914")
   void applyRequestMetadata(
```

Comment:
> but in either case I want to reserve the right to completely change the contents

Okay, but that isn't stable.  Leave the `@ExperimentalAPI` up top if thats what you want.  

> I don't want to provide that guarantee.  
If you remove the annotation, you provide that guarantee.  

>It is impossible to extend CallCredentials without touching ExperimentalApi, so I feel like your sentence is a word-game that is skewing reality.

This isn't a word game;  you just didn't understand.  Extendees of CallCredentials, after it has been unmarked ExperimentalApi, will expect their classes to not break.  Leaving ExperimentalApi methods on it implies they will break, because said methods may change or be removed.  That's contradicting signal, and __certain__ to confuse some users.  That causes __us__ future pain.  Do not cause us future pain.